tools/interactive/interactivetool_geoexplorer.xml

```diff
@@ -49,16 +49,16 @@ App: http://www.davesteps.com/geoExploreR/
 
 **Input data file MUST have as longitude column clomun 2, and as latitude column, column 3**
 
-Example input file (TAB separated)::
+Example input file (csv)::
```

Comment:
its still tab separated, isn't it?

Reply:
In fact no, it appears that it works with csv, maybe only but I need to try with tab

Reply:
Or you mean I have to modify the "aspect" of the example in the help section ?